core/src/main/kotlin/model/Documentable.kt

```diff
@@ -25,10 +26,10 @@ abstract class Documentable {
     override fun hashCode() = dri.hashCode()
 }
 
-data class PlatformDependent<out T>(
-    val map: Map<PlatformData, T>,
+data class SourceSetDependent<out T>(
+    val map: Map<SourceSetData, T>,
     val expect: T? = null
```

Comment:
I think that expect should point to `SourceSetData` containing `expect` element, not element itself.